x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_permissions.ts

```diff
@@ -200,7 +203,7 @@ export function getDataStreamPrivileges(dataStream: DataStreamMeta, namespace: s
   };
 }
 
-async function universalProfilingPermissions(packagePolicyId: string): Promise<[string, any]> {
+function universalProfilingPermissions(packagePolicyId: string) {
```

Comment:
could you add a return type to this and `apmPermissions`?

Reply:
Done